python_modules/dagster/dagster/_core/definitions/external_asset.py

```diff
@@ -99,6 +99,7 @@ def external_assets_from_specs(specs: Sequence[AssetSpec]) -> List[AssetsDefinit
         @multi_asset(
             specs=[
                 AssetSpec(
```

Comment:
asset specs don't have names?

Reply:
ah was meant to be on the multi_asset decorator